src/components/checkbox/index.tsx

```diff
@@ -122,7 +140,10 @@ class Checkbox extends Component<CheckboxProps, CheckboxState> {
     super(props);
 
     this.state = {
-      isChecked: new Animated.Value(this.props.value ? 1 : 0)
+      isChecked: new Animated.Value(this.props.value ? 1 : 0),
+      showError: false,
+      isValid: !(props.required && !props.value),
```

Comment:
Changing to `isValid: !(props.required && !props.value)` fails the tests

Reply:
I think I know why. If it is required and the value is false it isValid should be true since the user didn't call 'validate' yet and till it does, isValid is true

Reply:
It should be `!props.required || props.value` and not `!props.required || !props.value`

Reply:
Yes, that's what I did... sorry. Falling the tests.

Reply:
Test failed since this condition can returns value and value could be undefined. Fixed it by setting default value to false